### PR TITLE
Fix linting issues and workflows

### PR DESCRIPTION
**Pull request by hugihlynsson**
See title

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -64,8 +60,8 @@ const LandingPage: FunctionComponent = () => {
 
         <p className={styles.text}>
           Below are the key areas to consider when getting started with Avo. You
-          can start them in any order or in parallel, and skip those you don't
-          think apply to you.
+          can start them in any order or in parallel, and skip those you
+          don&apos;t think apply to you.
```
Comment (bjornj12) [inline pages/index.tsx:64]: This correct?

File: pages/index.tsx [hunk 2]
```diff
@@ -122,9 +118,9 @@ const LandingPage: FunctionComponent = () => {
         </p>
 
         <p className={styles.text}>
-          In the guide below you'll find information on what to consider before
-          importing, how to import, and our suggested next steps after the
-          import is completed.
+          In the guide below you&apos;ll find information on what to consider
```
Comment (bjornj12) [inline pages/index.tsx:121]: This correct?

Comment (bjornj12) [reply pages/index.tsx:121]: looks nice at least ✅

**Review by bjornj12 — APPROVED** [review]

File: pages/index.tsx (hunk 1 above)
Comment (hugihlynsson) [reply pages/index.tsx:64]: Yup, that's what the linter wants

File: pages/index.tsx (hunk 2 above)
Comment (hugihlynsson) [reply pages/index.tsx:121]: I think it is

**Pull request merged**